src/main/java/com/spotride/spotride/user/dto/UserRequestDto.java

```diff
@@ -0,0 +1,31 @@
+package com.spotride.spotride.user.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * User request DTO model.
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserRequestDto {
```

Comment:
Please make dto per request for Create and Update.

For Create id prop is not needed